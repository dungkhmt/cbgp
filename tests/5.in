20 20
20 30
12 5
10 0
7 2
17 10
8 5
15 11
15 0
17 9
19 7
16 5
8 2
7 18
4 9
7 1
9 14
18 0
13 0
6 2
7 3
14 17
18 1
11 10
12 4
5 2
2 0
13 4
14 11
12 13
5 13
18 2
1
0 0 21 1 20 3 30 0 14 6 16 4 21 4 27 0 18 5 13 7 9 7 11 9 15 5 12 4 12 8 10 10 21 9 11 7 20 0 23 4 
