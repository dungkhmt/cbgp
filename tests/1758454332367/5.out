(12, 5), (10, 0), (7, 2), (17, 10), (8, 5), (15, 11), (15, 0), (17, 9), (19, 7), (16, 5), (8, 2), (7, 18), (4, 9), (7, 1), (9, 14), (18, 0), (13, 0), (6, 2), (7, 3), (14, 17), (18, 1), (11, 10), (12, 4), (5, 2), (2, 0), (13, 4), (14, 11), (12, 13), (5, 13), (18, 2), 
numberIntersection=0 
minDistance=5.0 
minAngle=0.25732371497108897 
sumAngle=18.64805361318042 
sumDistance=18669.50467472786 
minEdgeLength=5.0990195135927845 
sumEdgeLength=705.8533815264934 

0.0000000000 5.0000000000 0.2573237150 18.6480536132 18669.5046747279 5.0990195136 705.8533815265
0: (0, 76),
1: (44, 75),
2: (44, 69),
3: (80, 80),
4: (23, 55),
5: (29, 66),
6: (80, 0),
7: (63, 80),
8: (36, 62),
9: (23, 47),
10: (12, 49),
11: (3, 38),
12: (25, 61),
13: (10, 69),
14: (14, 25),
15: (1, 61),
16: (51, 0),
17: (17, 48),
18: (25, 80),
19: (80, 11),
0 76 44 75 44 69 80 80 23 55 29 66 80 0 63 80 36 62 23 47 12 49 3 38 25 61 10 69 14 25 1 61 51 0 17 48 25 80 80 11 
0: (0, 76), 1: (44, 75), 2: (44, 69), 3: (80, 80), 4: (23, 55), 5: (29, 66), 6: (80, 0), 7: (63, 80), 8: (36, 62), 9: (23, 47), 10: (12, 49), 11: (3, 38), 12: (25, 61), 13: (10, 69), 14: (14, 25), 15: (1, 61), 16: (51, 0), 17: (17, 48), 18: (25, 80), 19: (80, 11), 
numberIntersection=0 minAngle=0.2573237149710881 sumAngle=18.648053613191802 minEdgeLength=5.0990195135927845 sumEdgeLength=705.8533815264991 minDistance=5.0 sumDistance=18669.504674728913 
Time taken: 8.223 s
